evm_arithmetization/src/cpu/kernel/constants/mod.rs

```diff
@@ -188,6 +193,10 @@ const HASH_CONSTANTS: [(&str, [u8; 32]); 2] = [
         "EMPTY_NODE_HASH",
         hex!("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),
     ),
+    (
+        "EMPTY_STRING_POSEIDON_HASH",
+        hex!("3baed9289a384f6c1c05d92b56c801c2d2e2a7050d6c16538b814fa186835c79"),
```

Comment:
Probably good to rename `EMPTY_STRING_HASH` into `EMPTY_STRING_KECCAK_HASH` all over the place for clarity / distinction.